app/code/community/Bolt/Boltpay/Helper/ConfigTrait.php

```diff
@@ -27,7 +27,9 @@ trait Bolt_Boltpay_Helper_ConfigTrait
      */
     public function isBoltPayActive()
     {
-        return Mage::getStoreConfigFlag('payment/boltpay/active');
+        return
+            Mage::getSingleton("boltpay/featureSwitch")->isSwitchEnabled(Bolt_Boltpay_Model_FeatureSwitch::BOLT_ENABLED_SWITCH_NAME)
```

Comment:
We definitely can do this in this way, but the problem is we have another idea in M2 and WooC: special helper for each feature switch in feature switch class.
I don't think that one approach is much better than another one.
But I think if we handle the same things differently in different plugins, we increase entropy and complicate further development.

But I don't mind merge this PR.

Reply:
This is the one line of code in one place approach.

Reply:
The version I know is Keep It Simple, Stupid 😄